accountKit.go

```diff
@@ -42,9 +45,13 @@ func NewAccountKit(appID, appSecret string) *AccountKit {
 	}
 }
 
-func (a *AccountKit) VerifyByToken(token string) (*Profile, error) {
+func (a *AccountKit) VerifyByToken(userAccessToken string, accountKitAppSecret string) (*Profile, error) {
```

Comment:
@hsinhoyeh yes, you are right. We do not need to pass in the `accountKitAppSecret` again, it can already be retrieved via `a.AppSecret`.  Resolved.